pie.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html>
+   <head>
+      <meta charset = "utf-8">
+      <title>Pie</title>
+      <link rel = "stylesheet" href = "index.css">
+   </head>
+   
+   <body style = "background-color: bisque;">
+      <header class = "MyHeader">
+         <div style="display:inline-block;vertical-align:top;">
+            <img src = "https://www.agileventures.org/assets/full_logo2_agile_ventures-06e917d2a31455091f5e4b74b7ca6592d2f8f5b4d5de52bdc9ea9a2505c6581e.png" width="100px">
+         </div>
+         <div style="display:inline-block;">
+            <h1>Cole's Curious Cuisines</h1>
+            
+         </div>
+         <nav class = "Nav">
+            <a href = "index.html">Home</a>
+            <a href = "aboutme.html">About the Chef</a>
+         </nav>
+         
+      </header>
+      <main>
+         <h1 id = tab>Blueberry Pie</h1>
+         <br />
+         <div class = "flexContainer">
+            <img id = "foodImage" src = "https://shewearsmanyhats.com/wp-content/uploads/2014/07/blueberry-pie-3.jpg" width = 300px>
+            <div class = "descContainer">
+               <p id = "foodDesc">Everybody's favorite pie done right. Yields 8 servings if you can stand the idea of sharing!</p>
+               <p id = "foodDesc">This household favorite blueberry pie acts as the perfect dessert. Serve topped with vanilla ice cream or eat on its own.</p>
+               <p id = "foodDesc">The perfectly delicious, not too sweet filling stuffed into a beautifully baked crispy, laticed pie satisfies your eyes before it ever even reaches your mouth.</p>
+            </div>
+         </div>
+         <br />
+         <div class = "Ingredients">
+               
+            <h1 id = tab>Ingredients</h1>
+         </div>
+         <ul>
+            <li id = "ingredientsList">pie dough for double 9 inch pie crust, chilled</li>
+            <li id = "ingredientsList">5 cups fresh blueberries</li>
+            <li id = "ingredientsList">1 tbsp freshly squeezed lemon juice</li>
+            <li id = "ingredientsList">2 tbsp milk or cream</li>
+            <li id = "ingredientsList">4 tbsp all purpose flour</li>
+            <li id = "ingredientsList">1/2 cup brown sugar</li>
+            <li id = "ingredientsList">1/4 tsp ground cinnamon</li>
+            <li id = "ingredientsList">1 large egg, beaten with 1 tbsp water to make egg wash</li>
```

Comment:
Putting an individual id on each list item tag works, but it would probably be much easier for you to just apply a style to the unordered or ordered list tag (depending on what you are trying to style). So I see you are only changing the font size, and this is a style that can easily be applied to the list parent tag. The way you have isn't wrong just could be easier :)